setup_analytics.py

```diff
@@ -14,7 +14,14 @@
 PostHog is developer-friendly, self-hosted product analytics. posthog-python is the python package.
 """
 
-install_requires = ["requests>=2.7,<3.0", "six>=1.5", "monotonic>=1.5", "backoff>=1.10.0", "python-dateutil>2.1"]
+install_requires = [
+    "requests>=2.7,<3.0",
+    "six>=1.5",
+    "monotonic>=1.5",
+    "backoff>=1.10.0",
+    "python-dateutil>2.1",
+    "distro>=1.5.0",  # Required for Linux OS detection in Python 3.9+
```

Comment:
https://github.com/PostHog/posthog-python/pull/198/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7R23

what's the difference between where I added it and this file 🙈

Reply:
We publish the `posthog` package as `posthoganalytics` so that it doesn't clash with the module name in our main Django backend. Most customers will use `posthog` which is where you changed it, we use `posthoganalytics` which this setup file relates to

Reply:
thank you, obvious once explained :)